internal/dashboard/handler/alarm_handler.go

```diff
@@ -47,6 +49,23 @@ func ListAlerts(ctx *gin.Context) ([]alert.Alert, error) {
 	return alarm.ListAlerts(ctx, filter)
 }
 
+// @ID DiagnoseAlert
+// @Tags Alarm
+// @Summary Diagnose alert
+// @Description Diagnose alert, trigger a job to do data collection and analization on background.
+// @Accept application/json
+// @Produce application/json
+// @Param body body alert.AnalyzeParam true "alert analyze param"
+// @Success 200 object response.APIResponse{data=[]job.Job}
```

Comment:
The response should be `job.Job` instead of `[]job.Job`

Reply:
oh yes, forgot to change this